python/lsst/pipe/analysis/utils.py

```diff
@@ -2309,3 +2366,44 @@ def corr2(ra, dec, g1a, g2a, g1b=None, g2b=None, raUnits="degrees", decUnits="de
         xy.process(catA, catB)
 
     return xy
+
+
+def measureRhoMetrics(rhoStat, thetaExtremum=1.0, operatorStr="<="):
+    """Convert Rho Statistics into a scalar metric.
+
+    Parameters
+    ----------
+    rhoStat : `treecorr.GGCorrelation` or `treecorr.KKCorrelation` object
+        A correlation function corresponding to a Rho statistic.
+    thetaExtremum : `float`, optional
+        Extremum value of angular scale (in units of arcmin) to consider
+        for averaging ``rhoStat``.
+    operatorStr : `str`, optional
+        An optional comparison operator to specify angular scales smaller than
+        or larger than ``thetaExtremum``.
+        Allowed values are "<=", ">=", "<" or ">".
+
```

Comment:
Document the `Raises` (https://developer.lsst.io/python/numpydoc.html#py-docstring-raises).